lib/hash.c

```diff
@@ -2388,6 +2415,7 @@ grn_hash_add(grn_ctx *ctx, grn_hash *hash, const void *key,
       id = grn_tiny_hash_add(ctx, hash, hash_value, key, key_size, value);
     }
     if (!id) {
+      ERR(GRN_UNKNOWN_ERROR, "[hash][add] failed to add id");
```

Comment:
`"[hash][add] failed to add: <%.*s>", key_size, key` is better.
(ID isn't added. We can get ID when adding is succeeded.)